### ISSUE TITLE
Chart

### ISSUE THREAD

**Issue by wojtulab** [konradkalemba/korona.ws#20]
Dużo osób szuka w necie wykresów.
Chodzi głównie wykres dzienny ilości testów do ilości testów pozytywnych.

Myślę że to bardzo ważna kwestia gdyż w każdym innym kraju robi ich się znacznie więcej. My już po agregacji zrobiliśmy dopiero te 5k, gdzie inne kraje w tym czasie miały już ponad 30k. Skala jest ogromna...

Myślałem o czymś na wzór tego plus ilość pozytywnych testów tj ilość zakażonych per dzień.

![Screenshot_20200315-155059](https://user-images.githubusercontent.com/14801148/76703842-ffdd1400-66d4-11ea-8cee-031c7eddfb00.jpg)

**Comment by konradkalemba** [comment]
Cześć, tak - więcej wykresów to coś, co chciałbym wprowadzić. Na razie jednak muszę znaleźć dla nich miejsce, ale dzięki za pomysł na wykres. 👍

**Comment by wojtulab** [comment]
Dziękuję.
Ja widzialbym to jako kolejna kolumnę obok:

mapa | statystyki | wykresy

Co do statystyk brakuje wartości procentowych o które bardzo dużo osób pyta na Twitterze np.
%wykrytych osób z testów, czyli
19/1019 (19 nowych zakażeń na 1019 testów) etc

**Comment by sosite** [comment]
@konradkalemba A może pływające kafelki (z przyciąganiem do krawędzi i siebie na wzajem) i możliwość ich włączania/wyłączania wg uznania? :)

**Comment by wojtulab** [comment]
jasne, zawsze to kolejny plus. Skad pobierasz dane?

**Comment by konradkalemba** [comment]
@sosite Też o tym myślałem - jak będę miał chwilę to się pobawię. :)
@wojtulab Uzupełniam swoją bazę danych o dane z twittera Minesterstwa Zdrowia.

**Comment by mulawamichal** [comment]
Myślę że zamiast na jedynym wykresie zarówno przypadki jak i zgony, lepiej by było na jednym wykresie potwierdzone przypadki nowe (slupki) oraz skumulowany (łamana), a na osobnym zgony (analogicznie)

**Comment by wojtulab** [comment]
Ja myślę że jest dobrze. Dodałbym jedynie słupki jako ilość testów dziennych, co zobrazuje na ile testów są te pozytywne.

**Comment by mulawamichal** [comment]
Teraz są dwie serie danych o skrajnie różnych rzędach wielkości. Na jednej liczymy w setkach (a pewnie zaraz peknie tysiąc), na drugiej pojedyncze przypadki (i oby tak zostało) . Przez to ta druga seria lezy praktycznie na osi wykresu i nie widać tam zadnych zmian

**Comment by wojtulab** [comment]
Może liczmy od 100 przypadków?

**Comment by Kamil-kozlowski** [comment]
@konradkalemba 
Można dodać pozostałe wykresy (np. testy pozytywne vs negatywne, przyrost hospitalizowanych) w formie zbliżonej do tej:

<img width="414" alt="image" src="https://user-images.githubusercontent.com/26530372/77789471-5e09df80-7063-11ea-8e38-8c00190a5bee.png">

Mógłbym się tym zająć.

**Comment by konradkalemba** [comment]
@Kamil-kozlowski Jasne, można by wykorzystywać inne dane, jednak zastanawiam się jak to rozwiązać pod względem UX/UI - obecnie mamy dosyć ograniczone miejsce. Zmniejszyłem, jak pewnie widziałeś, wykres z województwami, aby dać trochę "oddechu" mapie. Może, tak jak proponował @sosite, dodajmy opcję customizacji layoutu? Jeszcze jest taki "problem", że większość użytkowników (ok. 66%) wchodzi na stronę z urządzeń mobilnych - a tam też wydaje mi się, że można by popracować na układem.

**Comment by mulawamichal** [comment]
UI na komórce jest ok, ale na desktopie cos nie poszło:
![image](https://user-images.githubusercontent.com/44225214/77835799-ace17300-7150-11ea-8902-e356c7febe4e.png)
(ta czarna kreska to chyba "aktywna karta" z wykresami skumulowanymi, a jak sie kliknie obok niej, to pokazuje wykresy zwykłe (nieskumulowane))

**Comment by Kamil-kozlowski** [comment]
@mulawamichal trochę nie rozumiem, coś takiego da się osiągnąć jedynie jak na siłę zmniejszysz sobie wysokość okna przeglądarki i osiągniesz wymiary praktycznie uniemożliwiające komfortowe korzystanie z niej. Czy tak było?

**Comment by mulawamichal** [comment]
@Kamil-kozlowski nie, przeglądarka (Firefox)  otwarta normalnie w zmaksymalizowanym oknie.

**Comment by mulawamichal** [comment]
sprawdziłem na chrome i edge - z racji jakichś dodatkowych taskbarów obszar roboczy okna jest jeszcze niższy, i ramka z danymi "ogólnymi" wjeżdża jeszcze bardziej na wykres. edytowałem sobie styl żeby przesunąc wykresy w prawo - widac że góra okna jest schowana pod tą ramką na górze:
![image](https://user-images.githubusercontent.com/44225214/77894562-c03a2e80-7275-11ea-97e1-76011d68f4eb.png)

także trzeba chyba po prostu ograniczyć wielkośc tej ramki z głównymi danymi

**Comment by Kamil-kozlowski** [comment]
@mulawamichal tak, tak - zapomniałem napisać, że też odtworzyłem ten błąd u siebie. Faktycznie na ekranach ~13'' i mniejszych może coś takiego wystąpić.
@konradkalemba może można to zrobić przy pomocy jakichś Collapse Tabów "Pokaż liczby" i "Pokaż Wykresy", gdzie liczby byłyby domyślnie otwarte.
